pkg/commands/resolver.go

```diff
@@ -139,7 +139,10 @@ func makePublisher(po *options.PublishOptions) (publish.Interface, error) {
 			// TODO(jonjohnsonjr): I'm assuming that nobody will
 			// use local with other publishers, but that might
 			// not be true.
-			return publish.NewDaemon(namer, po.Tags), nil
+			return publish.NewDaemon(namer,
+				publish.WithLocalDomain(po.LocalDomain),
+				publish.WithLocalTags(po.Tags),
```

Comment:
I'd keep tags as a positional arg, just to avoid making a breaking change if we don't have to (unless I'm missing a benefit of making this an option).